src/dotnet-core-uninstall/dotnet-core-uninstall.csproj

```diff
@@ -37,11 +37,7 @@
   </ItemGroup>
 
   <Target Name="PublishSingleFile" AfterTargets="Build">
-    <MSBuild
-      BuildInParallel="False"
-      Projects="$(MSBuildThisFileDirectory)\dotnet-core-uninstall.csproj"
-      Properties="NoBuild=true"
-      Targets="Publish">
+    <MSBuild BuildInParallel="False" Projects="$(MSBuildThisFileDirectory)\dotnet-core-uninstall.csproj" Properties="NoBuild=true" Targets="Publish">
```

Comment:
Why is this changing in this PR?

Reply:
I think this was just caused by the formatting of the editor. The content is exactly the same.

Reply:
Please revert--the other way is more readable :)

Reply:
Done.